### PR TITLE
Add constants for bloom books(.bloompub)

### PR DESCRIPTION
**Pull request by nikkuAg**

### CODE REVIEW

File: le_utils/resources/formatlookup.json [hunk 1]
```diff
@@ -49,5 +49,8 @@
 	},
 	"zip": {
 		"mimetype": ".zip"
+	},
+	"bloompub": {
+		"mimetype": "application/blommpub+zip"
```
Comment (rtibbles) [inline le_utils/resources/formatlookup.json:54]: This appears to be a typo? `blommpub` instead of `bloompub`?

**Review by rtibbles — COMMENTED** [review]
One typo to correct. It also looks like because the spec has been updated, you will need to rerun `make build` and commit the result.

My last question is about whether we need to add a file format for `bloomd` as well, it's mentioned on the bloom-player NPM page, but not sure if there are files we actually need to integrate in that format too - any thoughts?

https://www.npmjs.com/package/bloom-player/v/2.2.39

**Comment by nikkuAg** [comment]
> My last question is about whether we need to add a file format for `bloomd` as well, it's mentioned on the bloom-player NPM page, but not sure if there are files we actually need to integrate in that format too - any thoughts?

I was initially confused about the .bloomd format, but from my exploration of the Bloom software, I found that files are most commonly saved in the .bloompub format. That's why I did not include the .bloomd format, as it is less commonly used.

**Comment by rtibbles** [comment]
This still needs `make build` to be run, and the resulting changes comitted and pushed to this PR.

File: le_utils/constants/file_formats.py [hunk 2]
```diff
@@ -94,6 +98,7 @@
     (H5P, "H5P"),
     (ZIM, "ZIM"),
     (EPUB, "ePub Document"),
+    (BLOOMPUB, "bloom Document"),
```
Comment (rtibbles) [inline le_utils/constants/file_formats.py:101]:
```suggestion
    (BLOOMPUB, "Bloom Document"),
```

**Review by rtibbles — APPROVED** [review]
Just suggested one more typo tweak.

**Pull request merged**